web/src/components/navbar/NavigatorLogo.tsx

```diff
@@ -10,14 +9,16 @@ interface Props {
 export const NavigatorLogo = (props: Props): ReactElement => {
```

Comment:
[sand] In an effort to rename, I think we should change the name of this component as well. Maybe `MyAccountLogo` or even just `Logo`? I dunno. Follow your heart.